### ISSUE TITLE
"sort current file alphabetically"

### ISSUE THREAD

**Issue by RiccardoCantoni** [micheletriaca/fast-sfdc#33]
command to reorder metadata files alphabetically to facilitate versioning

**Comment by micheletriaca** [comment]
Good idea. Unfortunately, doing it properly is slightly more involved than sorting a few XML keys.
Salesforce metadata has schema-defined element ordering, while some repeated elements are safe to sort and others are semantically ordered. We also need to handle formatting details such as indentation, line endings, and exactly one final newline—across both source and Metadata API formats, with their wonderfully inconsistent file extensions.
The right solution is therefore a schema-aware, byte-idempotent metadata formatter shared by sfdy and Fast, so that formatting, retrieving, converting, and rendering all produce the same output.
We’ll add this to the roadmap. In the meantime, we’ll separately fix any renderer-specific ordering issues that create unnecessary diffs.
